### PR TITLE
Update Plexus.csv

### PR DESCRIPTION
**Pull request by arnavpurani**
Updated 3 app scores to more recent reviews.

### CODE REVIEW

File: src/data/Plexus.csv [hunk 1]
```diff
@@ -20,7 +20,7 @@ Afterpay,November,2020,X,X,4,No reported issues
 Air NZ,June,2020,3,Requires disabling notification class,X,X
 Airbnb,November,2020,2,"Can't login, no mapview",X,X
 AirMessage,November,2020,4,No reported issues,X,X
-Airtel Thanks,October,2020,X,X,4,No reported issues
+Airtel Thanks,November,2020,4,No reported issues,X,X
```
Comment (tomkonidas) [inline src/data/Plexus.csv:23]:
```suggestion
Airtel Thanks,November,2020,4,No reported issues,4,No reported issues
```

File: src/data/Plexus.csv [hunk 2]
```diff
@@ -297,7 +297,7 @@ OfficeSuite Pro,September,2020,X,X,4,"Manually installed, unupgradeable"
 OKCupid,August,2020,1,Can't even login,X,X
 Olimpica Stereo,November,2020,X,X,4,No reported issues
 Omega Digi Bible,November,2020,3,Will not work in second phone user account,3,Will not work in second phone user account
-OneDrive,November,2020,X,X,1,Crashes on startup
+OneDrive,November,2020,4,No reported issues,X,X
```
Comment (tomkonidas) [inline src/data/Plexus.csv:300]:
```suggestion
OneDrive,November,2020,4,No reported issues,1,Crashes on startup
```

File: src/data/Plexus.csv [hunk 3]
```diff
@@ -457,7 +457,7 @@ Waze,November,2020,1,"Unusable-maps don't load",1,"Unusable-maps don't load"
 Wear OS by Google,November,2020,1,"Unusable without Google Play Services",X,X
 WeChat,May,2020,4,No reported issues,X,X
 Wells Fargo,November,2020,X,X,4,No reported issues
-WhatsApp,August,2020,3,No notifications unless the app is running in the background,4,No reported issues
+WhatsApp,November,2020,4,No reported issues,X,X
```
Comment (tomkonidas) [inline src/data/Plexus.csv:460]:
```suggestion
WhatsApp,November,2020,4,No reported issues,4,No reported issues
```

**Review by tomkonidas — CHANGES_REQUESTED** [review]
Hi, we should not change tested scores to untested. Just update the parts that are tested.

**Comment by arnavpurani** [comment]
Hi, sorry I misunderstood the instruction. Thanks.

**Pull request merged**